inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/single_layer/softmax.hpp

```diff
@@ -35,6 +36,12 @@ class SoftMaxLayerTest : public testing::WithParamInterface<softMaxLayerTestPara
 
 protected:
     void SetUp() override;
+    std::shared_ptr<ngraph::Function> makeSoftMax(const std::string& name = "");
```

Comment:
the same as for similar code in convolution

Reply:
As the same reason, it will be returned to the original function.

Reply:
reply as for convolution